ega-data-api-dataedge/src/main/java/eu/elixir/ega/ebi/dataedge/exception/FileNotAvailableException.java

```diff
@@ -0,0 +1,35 @@
+/*
+ *
+ * Copyright 2020 EMBL - European Bioinformatics Institute
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+package eu.elixir.ega.ebi.dataedge.exception;
+
+public class FileNotAvailableException extends Exception {
+    private final String fileId;
+
+    public FileNotAvailableException(String fileId) {
+        this.fileId = fileId;
+    }
+
+    public FileNotAvailableException(String fileId, Throwable cause) {
+        super(cause);
+        this.fileId = fileId;
+    }
+
+    public String getFileId() {
+        return fileId;
+    }
+}
```

Comment:
I hope EgaFileNotFoundException & FileNotAvailableException serves diff. purposes :-)

Reply:
`EgaFileNotFoundException` is when the file does not exist at all, `FileNotAvailableException` is when the file exists but we cannot serve it.